src/config/cache.js

```diff
@@ -43,7 +43,7 @@ module.exports = (env) => {
       }
     },
     test: {
-      maxAge: CACHE_MAX_AGE,
```

Comment:
I think caching in the test environment was making it really hard to track down changes to trackgroups!